test/test-options-cluster.js

```diff
@@ -0,0 +1,42 @@
+// Copyright IBM Corp. 2015. All Rights Reserved.
+// Node module: strong-supervisor
+// This file is licensed under the Artistic License 2.0.
+// License text available at https://opensource.org/licenses/Artistic-2.0
+
+var CPUS = require('os').cpus().length;
+var options = require('../lib/options');
+var tap = require('tap');
+
+tap.test('cluster option validation', function(t) {
+  t.test('valid values', function(t) {
+    check(['node', 'sl-run', '--cluster=10'], 10);
+    check(['node', 'sl-run', '--cluster', '-10'], -10);
+    check(['node', 'sl-run', '--cluster', 'CPU'], CPUS);
+    check(['node', 'sl-run', '--cluster=cpu'], CPUS);
+    t.end();
+
+    function check(argv, expect) {
+      var o = options.parse(argv);
+      t.equal(o.cluster.size, expect);
+    }
+  });
+
+  t.test('invalid values', function(t) {
+    check('--cluster=on', 'on');
+    check('--cluster=off', 'off');
+    check('--no-cluster', 'no');
+    t.end();
+
+    function check(val, msg) {
+      try {
```

Comment:
I'm curious why this isn't just `t.throws`...

_extra_ verbose example:
```js
function check(val, msg) {
  var err = {
    code: 1,
    message: 'Invalid cluster option: ' + msg,
  };
  var fn = _.bind(options.parse, ['node', 'sl-run', val]);
  t.throws(fn, err);
}
```

----
💭  hm..💡 !
A new operator or syntactic sugar for bind/partial/curry/whatever would be really handy. I bet I would use it way more often than I would use arrow functions..